node/middlewares/generateMiddlewares/generateRewriterRoutes.ts

```diff
@@ -21,16 +21,19 @@ const LIST_LIMIT = 300
 
 type RoutesByBinding = Record<string, Record<string, Route[]>>
 
-const createRoutesByBinding = (routes: Internal[], report: Record<string, number>, storeBindings: Binding[]) => {
+const createRoutesByBinding = (routes: Internal[], report: Record<string, number>, storeBindings: Binding[], disableRoutesTerm: string) => {
   const storeBindingsIds = storeBindings.map(({ id }) => id)
   return routes.reduce(
     (acc, internal) => {
       report[internal.type] = (report[internal.type] || 0) + 1
-      const validRoute =
+      let validRoute =
         !startsWith('notFound', internal.type) &&
         internal.type !== 'product' &&
         !internal.disableSitemapEntry &&
         storeBindingsIds.includes(internal.binding)
+      if (disableRoutesTerm) {
+        validRoute = validRoute && !internal.from.includes('/' + disableRoutesTerm)
```

Comment:
We're disabling routes with `/term`; Is this the expected behavior? I mean, if we should expect that the term is always preceded with a `/`, this needs to be informed in the property description.

Reply:
@danzanzini This was used to guarantee not catching the `term` within a part of the url, for example `/homepage/hello+{term}/...` - however, thinking about it, I do prefer a more general approach where we just look for the `term` and nothing else, leaving the responsibility for whoever creates the `term` to do it wisely. I will commit this change.
Thank you for the input.